app/qml/components/MMGpsDataText.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick
+import "."
+import ".."
+import lc 1.0
+import QtQuick.Layouts
+
+Item {
+    property string titleText: "Default title"
```

Comment:
I would remove "Default title" and add it as required property. Rename to `title` only?